### PR TITLE
[OFE-306] update sentry version

### PR DESCRIPTION
**Pull request by rodrigo-marchi-loggi**
- update sentry version to 0.23.0
- change sentry init
- jira ticket: https://loggidev.atlassian.net/browse/OFE-306
- sentry doc: https://docs.sentry.io/platforms/rust/

### CODE REVIEW

**Review by dianasabi — APPROVED** [review]

**Comment by denisefavila** [comment]
Did u test the code with this bump? Last time I tried to bump the sentry dependency to 0.21.0, I had some troubles. As long I remember, we had some major problem in compatibility.,The 0.18 was the last one that worked.

**Comment by rodrigo-marchi-loggi** [comment]
Yep, tested at nasdaq running on my local machine...

File: src/sentry.rs [hunk 1]
```diff
@@ -5,14 +5,13 @@ pub struct SentrySettings {
     pub dsn: String,
 }
 
-pub fn init_sentry(settings: SentrySettings) {
+pub fn init_sentry(settings: SentrySettings) -> sentry_client::ClientInitGuard {
+    info!("Starting Sentry integration");
     sentry_client::init((
         settings.dsn,
         sentry_client::ClientOptions {
-            release: sentry_crate_release!(),
```
Comment (davilaerte) [inline src/sentry.rs:12]: **doubt**: Why we need change this code here?

Comment (rodrigo-marchi-loggi) [reply src/sentry.rs:12]: because this function changed in the new version o sentry

**Comment by davilaerte** [comment]
It would be good to leave a description of this PR more explanatory about the purpose of these changes here. Maybe adding the activity link in Jira could be useful too.

File: src/sentry.rs (hunk 1 above)
Comment (davilaerte) [reply src/sentry.rs:12]: I understand, you have a link to about these new updates, perhaps applying this change. It would be nice to post it here too.

**Review by davilaerte — APPROVED** [review]
LGTM :rocket:

**Pull request merged**